routes/skus.rb

```diff
@@ -86,7 +86,7 @@ def self.registered(app)
       app.delete '/skus/:id/?' do
         # delete a sku
         sku = Sku[id: params[:id]]
-        sku.delete unless sku.nil?
+        sku.reassign_resources unless sku.name == 'default'
```

Comment:
As noted above, add the reassign to the delete method, and instead of just not doing anything for the default sku, it would be nice to send a message explaining that the default sku cannot be deleted.

Also, it is possible we don't need a default sku at all - resources can just not have one, right? It might be good to have a default one as a placeholder so no logic gets confused, but a nil in the sku field should not be hard to handle. If a default sku makes view logic simpler, keep it, otherwise lets not have one.